profiles/common.nix

```diff
@@ -14,6 +14,10 @@
   # Allow PMTU / DHCP
   networking.firewall.allowPing = true;
 
+  # Use networkd instead of the pile of shell scripts
+  networking.useNetworkd = true;
+  networking.useDHCP = false;
```

Comment:
Since this is also a desktop profile, how does this interact with network-manager?

Reply:
I will try it and report back

Reply:
Seems to be working fine. NetworkManager is running side-by-side and it still used to connect to the wifi, acquire DHCP, ...